integration/index.test.js

```diff
@@ -0,0 +1,79 @@
+const childProcess = require('child_process');
+const fetch = require('node-fetch');
+
+let samProcess;
+const port = 5000;
+let lambdaUrl = `http://127.0.0.1:${port}/`;
+function localApiStarted(processOutput) {
+  // Example output when local function started - "Running on http://127.0.0.1:5000/"
```

Comment:
`beforeEach` hook starts the function locally and waits for output that `sam` has started, so the tests can begin.